src/xian/storage.py

```diff
@@ -1,27 +1,25 @@
-import pathlib
+import constants as c

```

Comment:
change it to xian.constants as c 
for example it breaks my integration test because it cant find the constants package when executed from another dir

Reply:
Why does that break integration tests? Can't you adjust the tests? I think if you add the `constants.py` file to your test test environment it should be fine.

Reply:
Oh i see what you're saying. Fixed it now.